app/models/manager_refresh/inventory/core.rb

```diff
@@ -0,0 +1,41 @@
+module ManagerRefresh::Inventory::Core
```

Comment:
In here I would put __everything__ that can be refreshed, i.e. really all models.
`AutomationManager` and `CloudManager` define what _they_ need.

Not sure if I understand what you mean